app/models/user.js

```diff
@@ -3,6 +3,23 @@ const logger = require('../logger');
 const { databaseError } = require('../errors');
 const { dbErrorCodes } = require('../helpers');
 
+const handleError = genericMessage => error => {
+  logger.error(error);
+  const { message, errorFn } = dbErrorCodes[error.name] || { message: genericMessage };
+  if (errorFn) {
+    throw errorFn(`${genericMessage}. ${message}`);
+  }
+  logger.error(message);
+  throw databaseError(genericMessage);
+};
+
+const prepareResponse = response => {
+  if (Array.isArray(response)) {
+    return response.map(e => e.dataValues);
+  }
+  return response && response.dataValues;
```

Comment:
Maybe you can define this as a sequelize serializer, check this [section](https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#25--serializers) of out standars